.readthedocs.yml

```diff
@@ -5,12 +5,16 @@
 # Required
 version: 2
 
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
```

Comment:
nice. Thanks for taking care of the deprecation and making sure that the versions used to build the docs requirements files match now.